restaurant/src/assets/home.css

```diff
@@ -0,0 +1,45 @@
+.home-page {
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+  background-image: url("bg1.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  min-height: 100%;
+}
+.Homeheader {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 10px;
+  font-weight: 700;
+  font-size: 4.5rem;
+  color: rgba(46, 125, 50, 0.5);
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+.Homeinfo {
+  display: flex;
+  align-items: center;
+  font-size: 1.5rem;
+  padding-left: 25%;
+  padding-right: 25%;
+  color: rgba(46, 125, 50, 1);
```

Comment:
You dont need to use rgba if you are not using opacity values (set as 1 here). Its better to use hex codes for colors.